tests/test_autotuner.py

```diff
@@ -198,6 +198,7 @@ def test_at_w_mpi():
     # to perform the autotuning. Eventually, the result is complete garbage; note
     # also that this autotuning mode disables the halo exchanges
     op.apply(time=-1, autotune=('basic', 'destructive'))
+
```

Comment:
why. 

You should _always_ check the `Files Changed` panel after a push to a PR, as it's easier to see if you've pushed unwanted edits

Reply:
Yes. I always 'push' that kind of fixes towards then end of a PR, so I kinda deliberatley left it.